camus-api/pom.xml

```diff
@@ -1,23 +1,35 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
 
-  <parent>
-    <groupId>com.linkedin.camus</groupId>
-    <artifactId>camus-parent</artifactId>
-    <version>0.1.0-SNAPSHOT</version>
-  </parent>
+	<parent>
+		<groupId>com.linkedin.camus</groupId>
+		<artifactId>camus-parent</artifactId>
+		<version>0.1.0-SNAPSHOT</version>
+	</parent>
 
-  <artifactId>camus-api</artifactId>
-  <name>Camus API.</name>
-  <packaging>jar</packaging>
+	<artifactId>camus-api</artifactId>
+	<name>Camus API.</name>
+	<packaging>jar</packaging>
 
-  <dependencies>
-    <dependency>
-      <groupId>org.apache.hadoop</groupId>
-      <artifactId>hadoop-client</artifactId>
-    </dependency>
-  </dependencies>
+	<dependencies>
```

Comment:
Looks like you have 'tab' instead of 'white spaces' in this file.  The indentation doesn't seem right.